docs/Installation.md

```diff
@@ -58,67 +58,81 @@ In your IEM open the S7 Connector and launch the configurator.
 
 Add a data source:
 
-![S7 Connector Data Source](graphics/S7_Connector_Data_Source.PNG)
+![AdddatasourceS7](graphics/AdddatasourceS7.PNG)
 
 For this example the following tags are added:
 
-![s7_connector_config](graphics/S7_Connector_Configuration.PNG)
+- GDB.signals.tanksSignals.actLevel
+- GDB.signals.tankSignals.actTemprature
+- GDB.process.numberProduced
+- GDB.process.numberFaulty
+
+![S7tags](graphics/S7tags.PNG)
```

Comment:
Please either change the picture to one without the deploy operation is in progress message or switch this picture with the one after

Reply:
This changes has been done